packages/rs-drive-abci/.env.testnet

```
@@ -33,21 +33,21 @@ CORE_CHECK_TX_JSON_RPC_PASSWORD=password
 INITIAL_CORE_CHAINLOCKED_HEIGHT=1243
 
 # https://github.com/dashevo/dashcore-lib/blob/286c33a9d29d33f05d874c47a9b33764a0be0cf1/lib/constants/index.js#L42-L57
-VALIDATOR_SET_QUORUM_TYPE=llmq_25_67
-VALIDATOR_SET_QUORUM_SIZE=25
+VALIDATOR_SET_QUORUM_TYPE=6
+VALIDATOR_SET_QUORUM_SIZE=100
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VALIDATOR_SET_QUORUM_SIZE key should go before the VALIDATOR_SET_QUORUM_TYPE key

(UnorderedKey)